ui/pages/confirmation/confirmation.js

```diff
@@ -203,7 +203,10 @@ export default function ConfirmationPage() {
           <Box
             alignItems="center"
             marginTop={1}
-            padding={[1, 4, 4]}
+            paddingTop={1}
+            paddingRight={4}
+            paddingLeft={4}
+            paddingBottom={4}
             flexDirection={FLEX_DIRECTION.COLUMN}
           >
             <SiteIcon
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props